### DISCUSSION TITLE
KServe storage-initializer error....!

### DISCUSSION THREAD

**Discussion by HrushikeshGowda** [kubeflow/kfserving#2937] [Q&A] [answered]
Hello guys...!
I am following https://kserve.github.io/website/0.7/modelserving/storage/uri/uri/ documentation to serve Tensorflow model
Once I run the tensorflow-from-uri-gzip.yaml (given in documentation) It gives storage-initialiser error like

raise RuntimeError("URI: %s did not respond with any of following \'Content-Type\': " % uri +
RuntimeError: URI: https://raw.githubusercontent.com/tduffy000/kfserving-uri-examples/master/tensorflow/frozen/model_artifacts.tar.gz did not respond with any of following 'Content-Type': application/x-tar, application/x-gtar, application/x-gzip, application/gzip

Please help me solve.....Thanks.... 🙏

**Comment by nirmesh** [comment]
@HrushikeshGowda  I am also facing same issue any update?

**Reply by HrushikeshGowda** [reply, marked as answer]
Hi @nirmesh 

I guess it works if u use .zip instead of .gz (model_artifacts.zip).
